akka-actor/src/main/java/akka/annotation/ApiMayChange.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (C) 2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.annotation;
+
+import java.lang.annotation.*;
+
+/**
+ * Marks APIs that are meant to evolve towards becoming stable APIs, but are not stable APIs yet.
+ * 
+ * Evolving interfaces can change from one feature release to another release (i.e. 2.1 to 2.2) without up-front notice.
```

Comment:
I agree with @patriknw. What we're after is basically "-experimental" but within a stable module. We must be able to address feedback quickly and mark it stable in the next minor. E.g. thing comes in during 2.3.4, we iterate on it during 2.3.x, and promote it in 2.4.0 - no need to wait the full major cycle.